pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -228,31 +228,30 @@ func (pl *DefaultPreemption) SelectVictimsOnNode(
 // considered for preemption.
 // We look at the node that is nominated for this pod and as long as there are
 // terminating pods on the node, we don't consider this for preempting more pods.
-func (pl *DefaultPreemption) PodEligibleToPreemptOthers(pod *v1.Pod, nominatedNodeStatus *framework.Status) bool {
+func (pl *DefaultPreemption) PodEligibleToPreemptOthers(pod *v1.Pod, nominatedNodeStatus *framework.Status) (bool, string) {
 	if pod.Spec.PreemptionPolicy != nil && *pod.Spec.PreemptionPolicy == v1.PreemptNever {
-		klog.V(5).InfoS("Pod is not eligible for preemption because it has a preemptionPolicy of Never", "pod", klog.KObj(pod))
-		return false
+		return false, fmt.Sprintf("pod is not eligible for preemption because it has a preemptionPolicy of Never.")
```

Comment:
we need to be as succinct as possible, because event messages have a size limit. I suggest:

`not eligible for preemption due to preemptionPolicy=Never`

(no period)

And there is no need for `Sprintf` :)

Reply:
done